vaadin-date-picker-flow-parent/vaadin-date-picker-flow-integration-tests/src/test/java/com/vaadin/flow/component/datepicker/DatePickerLocaleIT.java

```diff
@@ -155,23 +154,23 @@ public void datePicker_setUnsupportedLocale_warningIsShown() {
         applyLocale(new Locale("th", "TH"));
 
         waitUntil(driver -> getWarningEntries().toString().contains(
-                "The locale is not supported, using default locale setting(en-US)."));
+                "The locale is not supported, using default format setting (ISO 8601)."));
     }
 
     @Test
-    public void datePicker_setUnsupportedLocale_defaultUSLocaleIsUsed() {
+    public void datePicker_setUnsupportedLocale_defaultISO8601FormatIsUsed() {
         picker.setDate(LocalDate.of(2018, Month.MAY, 3));
 
         List<Locale> unsupportedLocales = List.of(new Locale("ar", "SA"),
-                new Locale("th", "TH"), new Locale("fa"), new Locale("ks"));
+                new Locale("th", "TH"), new Locale("fa"));
```

Comment:
This omission is due to Kashmiri (`ks`) locale being parsed without a problem by the `toLocaleDateString(locale)` call in the connector.